config/samples/simple_tls_secret.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
```

Comment:
Maybe we can group this secret yaml with the mTLS configuration example example into another sub-folder sth like: `upstream_mtls/`

Reply:
I consider this secret to be reusable across future tasks as it's a very basic tls secret.  It's not something specific only for the upstream mtls feature.